packages/kbn-reporting/server/config_schema.ts

```diff
@@ -60,7 +60,7 @@ const CaptureSchema = schema.object({
 const CsvSchema = schema.object({
   checkForFormulas: schema.boolean({ defaultValue: true }),
   escapeFormulaValues: schema.boolean({ defaultValue: false }),
-  enablePanelActionDownload: schema.boolean({ defaultValue: false }), // unused as of 9.0
+  enablePanelActionDownload: schema.maybe(schema.boolean({ defaultValue: false })), // unused as of 9.0
```

Comment:
So we still allow the `enablePanelActionDownload` setting to be in the `yml` file?

Reply:
Yes. We do not want to block instances from startup if this setting exists. The extent of the breaking change is just to ignore the setting.